lib/langfuse-ecr-ecs-deployment-cdk-stack.ts

```diff
@@ -93,13 +92,7 @@ export class CdkLangfuseEcrEcsFargateCloudFrontDeploymentStack extends cdk.Stack
             dbServerSG: postgresStack.dbServerSG,
         };
 
-        // new CdkFargateWithVpcDeploymentStack(this, `${envTyped.APP_NAME}-${props.environment}-${props.deployRegion}-CdkFargateWithVpcDeploymentStack`, {
-        //     ...ecsStackProps,
-        //     stackName: `${envTyped.APP_NAME}-${props.environment}-${props.deployRegion}-CdkFargateWithVpcDeploymentStack`,
-        //     description: `Langfuse Fargate deployment stack for ${props.environment} environment in ${props.deployRegion} region.`,
-        // });
-
-        new CdkAppRunnerWithVpcDeploymentStack(this, `${envTyped.APP_NAME}-${props.environment}-${props.deployRegion}-CdkFargateWithVpcDeploymentStack`, {
```

Comment:
I would like to maintain it if possible, because of refers to langfuse-ecr-apprunner-deployment-cdk-stack.ts